### DISCUSSION TITLE
Auto-scaling works with Vercel?

### DISCUSSION THREAD

**Discussion by gittibeauty** [vercel/vercel#5145] [Help] [answered]
Hi all, this is my first post in the Vercel discussion forum. I've already tried out some simple apps and I'm really excited about the possibilities Vercel provides!

I'm working on a Shopify site that was recently on national TV and received a burst of massive traffic. Shopify help up really well, despite us having 40k+ active users on the site. I'm looking into the possibility building a custom frontend in React, and I'm curious about how Vercel autoscales when receiving sudden high traffic like this.

I've read through the docs and read a couple of articles, but I don't have a clear picture about how much traffic it would be able to handle, how fast it can autoscale, and whether this would affect the price at all. Does anyone know about this?

Thanks in advance :)

Zach

**Comment by paulogdm** [comment, marked as answer]
> I'm working on a Shopify site that was recently on national TV and received a burst of massive traffic. Shopify help up really well, despite us having 40k+ active users on the site. I'm looking into the possibility building a custom frontend in React, and I'm curious about how Vercel autoscales when receiving sudden high traffic like this.

This number should not be a problem at all. However, you may need an Enterprise plan to scale beyond the Pro plan. Please check the [Fair Use Policy](https://vercel.com/docs/platform/fair-use-policy).

> I've read through the docs and read a couple of articles, but I don't have a clear picture about how much traffic it would be able to handle, how fast it can autoscale, and whether this would affect the price at all. Does anyone know about this?

We are a Serverless platform with a built-in CDN. Scaling is not a concern in this type of environment infrastructure-wise. We are serving some sites that had tremendous traffic in the past couple of months, such as [fallguys.com](https://fallguys.com/).

**Reply by gittibeauty** [reply]
Thanks for your great response 👍 

I notice under the Fair Use Policy, it lists "Load Testing" as **not** fair use - does that mean I shouldn't use load testing to test my Vercel app, or does it mean I shouldn't be using the Vercel server as a load testing engine?

**Reply by paulogdm** [reply]
It means both. We only allow load tests for enterprise customers at the moment.
